### PR TITLE
Bc 2955 jwt update for shd

### PR DESCRIPTION
**Pull request by SevenWaysDP**
# Description

## Links to Tickets or other pull requests
- https://ticketsystem.dbildungscloud.de/browse/BC-2955

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,40 +4,41 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "watch": "nodemon --watch ./ --watch views/ --watch controllers/ --watch build/ --ext js,hbs ./bin/www",
+    "watch": "npx nodemon --watch ./ --watch views/ --watch controllers/ --watch build/ --ext js,hbs ./bin/www",
     "debug": "nodemon --inspect=0.0.0.0:9311 --watch controllers/ --watch helpers/ --watch views/ --ext js,hbs ./bin/www",
```
Comment (bischofmax) [inline package.json:8]: If nodemon is missing locally for the watch script it's also missing for the debug script. Npx should be added there as well.

The alternative would be to add nodemon to devDependencies. Is this an option here?

Comment (SevenWaysDP) [reply package.json]: readme says: `npm install -g nodemon gulp`

File: package.json [hunk 2]
```diff
@@ -10,34 +10,39 @@
     "startd": "echo 'Do you mean 'npm run watch'?'",
     "test": "mocha 'test/**/*.js' --recursive --timeout 60000 --exit"
   },
+  "engines": {
+    "node": "16",
```
Comment (dyedwiper) [inline package.json:14]: You have to run npm install again, because the engines are not yet in package-lock.json.

File: package.json (hunk 1 above)
Comment (dyedwiper) [reply package.json]: As you updated a lot more than jsonwebtoken anyway, couldn't you add nodemon and gulp as dependencies in package.json and remove this instruction from the readme? Or if you don't wanna do it in this ticket, shall we make a new small ticket for this change? I believe it's a rather bad practice to install them globally.

**Review by CeEv — APPROVED** [review]

**Review by dyedwiper — APPROVED** [review]

**Pull request merged**